log4j-core/src/main/java/org/apache/logging/log4j/core/util/SystemClock.java

```diff
@@ -16,10 +16,15 @@
  */
 package org.apache.logging.log4j.core.util;
 
+import java.time.Instant;
+import org.apache.logging.log4j.core.time.MutableInstant;
+import org.apache.logging.log4j.core.time.PreciseClock;
+
 /**
  * Implementation of the {@code Clock} interface that returns the system time.
+ * @since 2.25
```

Comment:
The class itself is older, so we should remove the `@since`.

Reply:
Sounds good!